notifypy/os_notifiers/windows.py

```diff
@@ -84,13 +91,21 @@ def send_notification(
         notification_subtitle,
         notification_icon,
         application_name,
+        notification_audio,
     ):
         generated_file = self._generate_notification_xml(
             notification_title=notification_title,
             notification_subtitle=notification_subtitle,
             notification_icon=notification_icon,
             application_id=application_name,
+            notification_audio=notification_audio,
         )
+
+        if notification_audio:
+            winsound.PlaySound(
+                notification_audio, winsound.SND_ASYNC | winsound.SND_FILENAME
```

Comment:
```python
        if notification_audio:
            subprocess.Popen(
                [
                    "Powershell",
                    f'(New-Object Media.SoundPlayer "{notification_audio}").playsync()'
                ]
            )
```

At least for me, this command makes sounds more consistent, also fixes that issue you had with the imports.

Reply:
looks good to me! And getting rid of that import is always great.